src/locales/en.json

```diff
@@ -552,7 +552,7 @@
   "nft_claim_NFT_name": "\"{name}\"",
   "nft_claim_NFT_publisher": "Publisher",
   "nft_claim_title": "{name} and {count} other books",
-  "nft_claim_welcome_button_claim": "Login/Register",
+  "nft_claim_welcome_button_claim": "Next Step",

```

Comment:
```suggestion
  "nft_claim_welcome_button_claim": "Continue",

```

Reply:
[💬 Update cta button text in welcome page](https://github.com/likecoin/liker-land/pull/1732/commits/3c281af29ddca9418a2b16b576808fffb913db16)